core/slim_sim.cpp

```diff
@@ -6099,12 +6099,17 @@ void SLiMSim::DerivedStatesToAscii(tsk_table_collection_t *p_tables)
 void SLiMSim::AddIndividualsToTable(Individual * const *p_individual, size_t p_num_individuals, tsk_table_collection_t *p_tables, uint32_t p_flags)
 {
     // We use currently use this function in two ways, depending on p_flags:
-    //  1. (SLIM_TSK_INDIVIDUAL_REMEMBERED) to retain individuals to be forever remembered, or
-    //  2. (SLIM_TSK_INDIVIDUAL_ALIVE) to retain the final generation in the tree sequence.
-    // So, in case (1) we set the REMEMBERED flag,
-    // and in case (2) we set the ALIVE flag.  Individuals who are permanently
-    // remembered but still alive when the tree sequence is written out will
-    // have this method called on them twice, first (1), then (2), so they get both flags set.
+    //  1. (SLIM_TSK_INDIVIDUAL_REMEMBERED) for individuals to be permanently
+    //      remembered, or
+    //  2. (SLIM_TSK_INDIVIDUAL_RETAINED) for individuals to be retained only while
+    //      some of their genome (i.e. nodes) exists in the tree sequence ancestry, or
```

Comment:
remove the word "ancestry" (since the *ancestry* of nodes that are along edges is still in the tree sequence, but it's only the nodes at coalescence points that are written down)

Reply:
Oh, yes, that's true isn't it. Damn. I imagine that in some circumstances we might also want to retain unary nodes, and the individuals that have them. I guess this would be an option to `treeSeqSimplify()` (`keep_unary`) and also something that you could set for the automatic simplification. But that's another issue - I'll open it up now.

Reply:
Are these orthogonal issues, in fact?  Let's try to envision the ideal final design for this stuff, not build the API by incremental accretion.  In the ideal final design, is the `permanent=F` flag proposed here as a way to retain individuals rather than permanently remember them still the right API design?

Reply:
I just started discussion of this in #139; that's a better place than here, but let's resolve that discussion before finalizing this PR to make sure we're not painting ourselves into a corner somehow.